README.md

```diff
@@ -99,7 +104,10 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 - Handle labeled configurations
 - Lock/unlock configuration settings
 
-### � Azure CLI Extension
+### ☸️ Azure Kubernetes Service (AKS)
+- List AKS clusters in a subscription
```

Comment:
This one should be moved to the list starting on line 61 or even removed if we think it's redundant.